var/spack/repos/builtin/packages/py-melissa-core/package.py

```diff
@@ -37,6 +39,8 @@ class PyMelissaCore(PythonPackage):
     depends_on("py-python-rapidjson@1.8:", type="run")
     depends_on("py-scipy@1.10.0:", type="run")
     depends_on("py-cloudpickle@2.2.0:", type="run")
```

Comment:
Is this needed for sc23? Isn't there a missing plotext dep?

Reply:
Hi @adamjstewart, sorry for the delay. 

This is exact, I have created a [PR](https://github.com/spack/spack/pull/39023) for this dependency. I while update the recipe once it is accepted.

Reply:
Cloudpickle still isn't needed for sc23

Reply:
I fixed that one too, thanks for your thoroughness.